src/app/profile/views/ens_manager.nim

```diff
@@ -209,32 +209,36 @@ QtObject:
   proc registerENSGasEstimate(self: EnsManager, ensUsername: string, address: string): int {.slot.} =
     var success: bool
     let pubKey = status_settings.getSetting[string](Setting.PublicKey, "0x0")
-    result = registerUsernameEstimateGas(ensUsername, address, pubKey, success)
-    if not success:
-      result = 325000
+    try:
+      result = registerUsernameEstimateGas(ensUsername, address, pubKey, success)
```

Comment:
Shouldn't you check the `success` value anyway or now it always returns correctly?